acceptance-tests/src/test/java/org/hyperledger/besu/tests/acceptance/crosschain/getinfo/generated/Ctrt3Int.java

```diff
@@ -0,0 +1,170 @@
+/*
```

Comment:
When the .sol file for this is an interface, this file will not be generated.

Reply:
THanks.